integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -316,7 +317,9 @@ func BuildAddDONArgs(
 		if err2 != nil {
 			return nil, err2
 		}
-		signers, transmitters, configF, _, offchainConfigVersion, offchainConfig, err2 := ocr3confighelper.ContractSetConfigArgsForTests(
+		signers, transmitters, configF, _, offchainConfigVersion, offchainConfig, err2 := ocr3confighelper.ContractSetConfigArgsDeterministic(
```

Comment:
@krehermann this is the new libocr function. Its on us to supply common secrets shared across signers.